tests/semantic/subsumption_multiple_rules/subsumption_multiple_rules.dl

```diff
@@ -85,3 +85,21 @@ F(x1) <= F(x2) :-
    x1 <= x2.
 
 .output F
+
+// build full comparator for all indices in a relation
+
+.decl G(from:number, to:number)
+G(1, 1).
+
+.decl R(a: number, b: number)
+R(2,2).
+
+.decl ES(x:number, y:number) btree_delete
```

Comment:
I thought the btree_delete keyword was no longer required ?

Reply:
Well - we still have it; if there is a subsumptive rule, `btree_delete` will be automatically set.